cmake/branding/CMakeLists.txt

```diff
@@ -0,0 +1,196 @@
+# Colors and converts .svg files to .png files
+# Also creates platform-specific themed files (.ico, icns)
+#
+# Copyright (c) 2019, Tres Finocchiaro, <tres.finocchiaro@gmail.com>
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+
+FIND_PACKAGE(ImageMagick COMPONENTS convert)
+# Prefer COMMAND over PKG_CHECK_MODULES per https://apple.stackexchange.com/q/169601/147537
```

Comment:
Can we try `PKG_CHECK_MODULES` first and then fallback to `EXECUTE_PROCESS` instead?

Reply:
It fails in one of our environments, and I've documented it.  What else do you want?

Reply:
Due to lack or reply, I assume this conversation is resolved.